Terminal.Gui/View/View.cs

```diff
@@ -1395,7 +1395,8 @@ public Point ScreenToView (int x, int y)
 		public Point ScreenToBounds (int x, int y)
 		{
 			if (SuperView == null) {
-				return new Point (x - Frame.X + GetBoundsOffset ().X, y - Frame.Y + GetBoundsOffset ().Y);
+				var boundsOffset = GetBoundsOffset ();
```

Comment:
I assume you did this to ease debugging? I often do the same.

I believe (but am not 100%) sure that there's no performance penalty for calling `GetBoundsOffset` twice in `return new Point (x - Frame.X + GetBoundsOffset ().X, y - Frame.Y + GetBoundsOffset ().Y);` because the compiler will optimize it.

Reply:
Was due the performance to not call twice.

Reply:
Do you know for sure it's actually a perf hit? 

I'm not asking you to change it, I'm just very curious.

Reply:
FYI: https://stackoverflow.com/a/2162893/297526

My favorite part of his answer:

> n short, you cannot possibly know whether writing the code to cache the result rather than recomputing it is actually (1) faster, or (2) better performing. 

IMO, in this situation, we don't even know if this codepath is a critical perf vector. Until we do, my feeling is we should optimize for code readability, testability, and debugabillity.  

I don't think either version makes any difference for readabilty or testability, but I do think your version is easier to debug. So yay. :-)